.github/workflows/test.yml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

```suggestion
        python-version: [3.8, 3.9, 3.10, 3.11]
```

As I see in the [documentation](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#using-a-matrix-strategy), it is not required to use "". Do you use here for any reason? Maybe I misunderstood documentation...

---

Yes, the reason is because 3.10 without brackets is parsed as 3.1 for some reason.